changelog.json

```diff
@@ -1,5 +1,28 @@
 {
   "releases": {
+    "1.3.0": [
+      "[New] Notification displayed in History tab when the base branch moves ahead of the current branch - #4768",
+      "[New] Repository list displays uncommitted changes count and ahead/behind information - #2259",
+      "[Added] Option to move repository to trash when removing from app - #2108. Thanks @say25!",
+      "[Added] Syntax highlighting for PowerShell files - #5081. Thanks @say25!",
+      "[Fixed] \"Discard Changes\" context menu discards correct file when entry is not part of selected group - #4788",
+      "[Fixed] Display local path of selected repository as tooltip - #4922. Thanks @yongdamsh!",
+      "[Fixed] Display something when repository is located at drive root - #4924",
+      "[Fixed] Ensure commit message is cleared after successful commit - #4046",
+      "[Fixed] Handle legacy macOS right click gesture - #4942",
+      "[Fixed] History omits latest commit from list - #5243",
+      "[Fixed] Markdown header elements hard to read in dark mode - #5133. Thanks @agisilaos!",
+      "[Fixed] Only perform ahead/behind comparisons when branch selector is open - #5142",
+      "[Fixed] Relax checks for merge commits for GitHub Enterprise repositories - #4329",
+      "[Fixed] Render clickable link in \"squash and merge\" commit message - #5203. Thanks @1pete!",
+      "[Fixed] Return key disabled when no matches found in Compare branch list - #4458",
+      "[Fixed] Selected commit not remembered when switching between History and Changes tabs - #4985",
+      "[Fixed] Selected commit when comparing is reset to latest when Desktop regains focus - #5069",
+      "[Fixed] Support default branch detection for non-GitHub repositories - #4937",
+      "[Improved] Change primary button color to blue for dark theme - #5074",
+      "[Improved] Diff gutter elements should be considered button elements when interacting - #5158",
+      "[Improved] Repository list badge style tweaks and tweaks for dark theme - #5095",
```

Comment:
same as above. 🍄

Reply:
Since Repo List is brand new for production, I'm not sure there's value in having an improved line item for it.

Reply:
There's a `New` entry for the feature already:

> "[New] Repository list displays uncommitted changes count and ahead/behind information - #2259"

This entry is for follow-up dark theme tweaks, which I think is unnecessary duplication. I'll move the issue link into the original item.